testapi.pm

```diff
@@ -1049,7 +1049,16 @@ and can be tweaked by setting the C<$wait> positional parameter.
 =cut
 
 sub script_output {
-    return $distri->script_output(@_);
+    my $script = shift;
+    my %args   = testapi::compat_args(
+        {
+            timeout            => undef,
+            proceed_on_failure => 0,                                         # fail on error by default
+            quiet              => testapi::get_var('_QUIET_SCRIPT_CALLS'),
+            type_command       => length($script) < 80,
```

Comment:
I would let the defaults up to the distri implementation for that option.
```suggestion
            type_command       => undef,
```

Reply:
done